CMakeLists.txt

```diff
@@ -71,6 +72,17 @@ add_test (NAME qrack_accuracy
     COMMAND accuracy
     )
 
+# Declare the OCL precompilation executable
+add_executable (qrack_cl_precompile
```

Comment:
Should this be gated on whether or not OpenCL is supported?

Reply:
Probably not, since we install OpenCL-related headers. We install everything else, regardless.